pegasus/sites.v3/code.org/views/hoc_2024_music_lab_jam_hero.haml

```diff
@@ -0,0 +1,29 @@
+:ruby
+  hoc_mode = DCDO.get('hoc_mode', CDO.default_hoc_mode)
+
+-# TODO ACQ-2556 - Remove this after 2024 HOC
+%link{href: "/css/generated/hoc-2024-music-lab-jam-hero.css", rel: "stylesheet"}
+
+.hoc-2024-music-lab-jam-hero
+  .wrapper
+    .content-wrapper
+      %p.body-one.white
+        - if hoc_mode == "soon-hoc"
+          =hoc_s(:social_hoc_is_coming)
+        - elsif hoc_mode == "actual-hoc"
+          =hoc_s(:social_hoc2018_hoc_here)
```

Comment:
Here's how it looks when `hoc_mode` is set to `actual-hoc`:

![localhost code org_3000_(Code org - 1366x768)](https://github.com/user-attachments/assets/4e1d3e2a-e535-49d1-8e53-f19c722bac0e)